### PR TITLE
Patching cluster config to enable logging (PLAT-2262)

### PR DESCRIPTION
**Pull request by ddl-denis-parnovskiy**
Use AwsCustomResource to execute update function

### CODE REVIEW

**Review by Secretions — COMMENTED** [review]
Any chance we could piggyback on this and also have it tag the eks cluster?

We're not tagging it because there is no cloudformation api for tagging eks cluster (see: https://github.com/aws/aws-cdk/issues/4995), so one of the things I wanted to create a big provisioning lambda to do is work around this lack of tagging.

**Comment by ddl-denis-parnovskiy** [comment]
> Any chance we could piggyback on this and also have it tag the eks cluster?

We cannot just add tags to this function call. 

But there is a good chance to use the same approach to call tagging API. 

AwsCustomResource is converted into JS call.

JS SDK syntax is as follows:
```
var params = {
  ResourceARNList: [ /* required */
    'STRING_VALUE',
    /* more items */
  ],
  Tags: { /* required */
    '<TagKey>': 'STRING_VALUE',
    /* '<TagKey>': ... */
  }
};
resourcegroupstaggingapi.tagResources(params, function(err, data) {
  if (err) console.log(err, err.stack); // an error occurred
  else     console.log(data);           // successful response
});
```

What I do not know here is the relation between "resourcegroupstaggingapi" and `service` field. An experiment could answer this. Or not :)

I can take it.

**Comment by ddl-denis-parnovskiy** [comment]
Changing to WIP because I want to add "on_delete" call to delete the logging group on cluster tear down. Now it is not deleted automatically because it is not part of CF stack.

**Review by Secretions — APPROVED** [review]

File: cdk/domino_cdk/provisioners/eks/eks_cluster.py [hunk 1]
```diff
@@ -53,6 +57,33 @@ def provision(
...
+            "UpdateClusterConfigCustom",
+            timeout=cdk.Duration.minutes(10),  # defaults to 2 minutes
+            log_retention=logs.RetentionDays.ONE_DAY,  # defaults to never delete logs
+            policy=cr.AwsCustomResourcePolicy.from_sdk_calls(resources=cr.AwsCustomResourcePolicy.ANY_RESOURCE),
+            on_create={
+                "service": "EKS",
+                "action": "updateClusterConfig",
+                "parameters": params,
+                "physical_resource_id": cr.PhysicalResourceId.of("UpdateClusterConfigCustom"),
+                "ignore_error_codes_matching": "...",
```
Comment (Secretions) [inline cdk/domino_cdk/provisioners/eks/eks_cluster.py:83]: A comment explaining why this is here might not be a bad idea.

**Pull request merged**